lib/inferno/apps/web/index.html.erb

```diff
@@ -9,10 +9,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta id="base-path" name="base-path" content="<%= Inferno::Application['base_path'] %>">
-    <meta
-      name="description"
-      content="FHIR Testing"
-    />
+    
+    <!-- Social media link unfurling meta tags -->
+    <meta name="application-name" content="Inferno" />
+    <meta name="og:type" content="website" />
+    <meta name="og:site_name" content="Inferno" />
+    <meta name="twitter:card" content="summary" />
+    
```

Comment:
I'm really curious if sites will load JavaScript up, render it, and then look at the meta tags... or just look at the html of the original page.  If it just looks at html and doesn't try the Javascript, I think just adding the inferno logo image (no text) here and 'Inferno Test Session' for the title would be helpful.

Reply:
For some sites, html will just be pulled -- I think this is a bit unavoidable unless we do some backend handling.

Reply:
Alright, I think this is worth a go to see how it looks in practice for the common sharing spots.  If you have time and interest you could check out simplifying this and making it more robust via backend html rendering, but if you have neither and this works I'm very much fine with this approach.